### ISSUE TITLE
File type mismatch (c#)

### ISSUE THREAD

**Issue by Lap1n** [hrsh7th/vim-vsnip#174]
Hi,

When I try to import vs code snippets for C# it does't work out of the box. After investigation, it turns out that VS code identifies the C# language with "csharp" while vim identifies it with the "cs" keyword. One quick fix is to rename the language config to 'cs' in the package.json of the snippet project, but it seems like an ugly hack. Is there a better way to handle this case?

I've tried the following approach but it did not work : 
```vimscript
let g:vsnip_filetypes = {}
let g:vsnip_filetypes.csharp = ['cs']
```

Maybe I missed something?

Thanks for this great plugin.

**Comment by hrsh7th** [comment]
It related to https://github.com/hrsh7th/vim-vsnip/blob/master/autoload/vsnip/source/vscode.vim#L95
I will add it.

**Comment by hrsh7th** [comment]
Fixed by 925fb3d0163e4c50b8d639029837696ead0f5a56 THank you.
